docs/sphinx/_stardoc_footer.md

```diff
@@ -7,6 +7,8 @@
 [`Label`]: https://bazel.build/rules/lib/Label
 [`list`]: https://bazel.build/rules/lib/list
 [`str`]: https://bazel.build/rules/lib/string
+[str]: https://bazel.build/rules/lib/string
```

Comment:
Why do we have both? `str` and `str`?

Reply:
The "ref" style links require an exact match of their contents. For whatever reason (I'll go with "whimsy"), I made it output

```
list of [str]
# instead of
list of [`str`]
```

(I don't really like either style; `list[str]` is more natural IMHO, but I'm not actually sure how to make it render with code font, clickable type links, and look OK)